tests/ociswrapper/ocis/ocis.go

```diff
@@ -105,7 +120,11 @@ func WaitForConnection() bool {
 			req.Header.Set("X-Request-ID", "ociswrapper-"+strconv.Itoa(int(time.Now().UnixMilli())))
 
 			res, err := client.Do(req)
-			if err != nil || res.StatusCode != 200 {
+			if err != nil || res.StatusCode >= 400 {
+				if res != nil {
+					log.Println(fmt.Sprintf("oCIS server up but request failed: %v", res.StatusCode))
+					return true
```

Comment:
Not sure if we want to `return true` here. Isn't it better if we wait until we a get clean `200` so we can be sure server is ready?

Reply:
I can easily do 200 check. go for it?

Reply:
I'd rather adjust the request then. So yes let's go for `200` :+1: